test/rdbx_driver.c

```diff
@@ -348,6 +348,9 @@ double rdbx_check_adds_per_second(int num_trials, unsigned long ws)
             ++failures;
     }
     timer = clock() - timer;
+    if (timer < 1) {
```

Comment:
This and the one other place where `t = clock() -t` is done will prevent a possible divide by zero.  However, it's possible that clock() might return -1 if there is an error.  Do we want to just return call exit() if clock will not function properly?

Reply:
Potentially